### PR TITLE
reduce route files to hdc with 3 variables

### PR DESCRIPTION
**Pull request by whitfield-mj**
Interested to hear opinions on this

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -221,12 +219,8 @@ module.exports = function createApp({
 
     app.use(['/caseList/', '/'], createCaseListRouter({logger, caseListService, authenticationMiddleware}));
     app.use('/hdc/taskList/',
-
         createTaskListRouter({logger, prisonerService, licenceService, authenticationMiddleware}));
-    app.use('/hdc/eligibility/', createEligibilityRouter({logger, licenceService, authenticationMiddleware}));
-    app.use('/hdc/proposedAddress/', createProposedAddressRouter({logger, licenceService, authenticationMiddleware}));
-    app.use('/hdc/licenceConditions/', createLicenceConditionsRouter({
-        logger, licenceService, conditionsService, authenticationMiddleware}));
+    app.use('/hdc/', createHdcRouter({logger, licenceService, conditionsService, authenticationMiddleware}));
```
Comment (romidane) [inline server/app.js]: I like it

File: server/routes/hdc.js [hunk 2]
```diff
@@ -72,22 +77,24 @@ module.exports = function({logger, licenceService, conditionsService, authentica
...
         logger.debug(`GET licenceConditions/${formName}/${nomisId}`);
 
         const rawLicence = await licenceService.getLicence(nomisId);
         const {licenceSection, nextPath, licenceMap} = formConfig[formName];
-        const dataPath = licenceMap || ['licence', 'licenceConditions', licenceSection];
+        const dataPath = licenceMap || ['licence', sectionName, licenceSection];
         const data = getIn(rawLicence, dataPath) || {};
 
-        res.render(`licenceConditions/${formName}Form`, {nomisId, data, nextPath});
+        res.render(`${sectionName}/${formName}Form`, {nomisId, data, nextPath});
```
Comment (romidane) [inline server/routes/hdc.js]: Like the name change

Comment (romidane) [inline server/routes/hdc.js]: we still have to think about maybe moving `${formName}Form` to the config?

Comment (whitfield-mj) [reply server/routes/hdc.js]: Agreed

**Pull request merged**